src/main/java/data/Database.java

```diff
@@ -1,30 +1,35 @@
 package data;
 
+import java.io.File;
 import java.io.FileInputStream;
 import java.io.FileNotFoundException;
 import java.io.FileOutputStream;
 import java.io.InputStream;
 import java.io.OutputStream;
-import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 /* This class creates an instance of a database that a file can be written to and read from.
 Assumptions: for each runID, only 1 of each datatype can be associated with it. */
 public class Database {
-  /* Place where files are stored.*/
-  private String directory;
 
-  /* List of all files stored in Database.*/
-  private ArrayList<String> files;
+  /* Folder for Database. */
+  File folder;
 
   /* Creates instance of a database with this name. */
   public Database(String name) {
-    this.directory = name;
-    files = new ArrayList<String>();
+    folder = new File(name);
+    folder.mkdir();
   }
 
   /* Returns name associated with Database. */
-  public String getDatabase() {
-    return directory;
+  public File getDatabase() {
```

Comment:
this method is functionally redundant to the getDirectoryName() method. will delete